<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>563</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Settings_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/cogs.png</normaloff>../images/cogs.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>40</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="Status_Label">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Name_Label">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 24pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Number_Label">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="Answer_Button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background-color: rgba(38, 162, 105, 255);
gridline-color: rgb(198, 70, 0);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border: 0px;
border-radius: 5px;}
QPushButton:focusressed {background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(38, 162, 105, 255), stop:1 rgba(15, 63, 42, 255));
gridline-color: rgb(198, 70, 0);
font: 16pt &quot;Ubuntu&quot;;
color: rgba(38, 162, 105, 255);
border: 0px; }
 QPushButton:focus {background-color: rgba(255, 255, 255, 255);
gridline-color: rgb(198, 70, 0);
font: 16pt &quot;Ubuntu&quot;;
color: rgba(38, 162, 105, 255);
border: 0px; 
border-radius: 5px;}
</string>
       </property>
       <property name="text">
        <string>Answer</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Call_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Hang_Button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background-color: rgb(246, 97, 81);
gridline-color: rgb(198, 70, 0);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border: 0px;
border-radius: 5px;}
QPushButton:focusressed {background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(38, 162, 105, 255), stop:1 rgb(255, 255, 255););
gridline-color: rgb(198, 70, 0);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(246, 97, 81);
border: 0px; }
 QPushButton:focus {background-color: rgb(255, 255, 255);
gridline-color: rgb(198, 70, 0);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(246, 97, 81);
border: 0px; 
border-radius: 5px;}
</string>
       </property>
       <property name="text">
        <string>Decline</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="Main_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background-color: rgba(255, 255, 255, 255);
font: 12pt &quot;Ubuntu&quot;;
border: 0px;}
QPushButton:focuspressed {background-color: rgba(255, 255, 255, 255);
gridline-color: rgb(198, 70, 0);
font: 12pt &quot;Ubuntu&quot;;
color:black;
border: 0px; }
 QPushButton:pressed {background-color: rgba(150, 150, 150, 255);
gridline-color: rgb(198, 70, 0);
font: 12pt &quot;Ubuntu&quot;;
color:black;
border: 0px; }</string>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/indicator.png</normaloff>../images/indicator.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="History_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background-color: rgba(255, 255, 255, 255);
font: 12pt &quot;Ubuntu&quot;;
border: 0px;}
QPushButton:focuspressed {background-color: rgba(255, 255, 255, 255);
gridline-color: rgb(198, 70, 0);
font: 12pt &quot;Ubuntu&quot;;
color:black;
border: 0px; }
 QPushButton:pressed {background-color: rgba(150, 150, 150, 255);
gridline-color: rgb(198, 70, 0);
font: 12pt &quot;Ubuntu&quot;;
color:black;
border: 0px; }</string>
       </property>
       <property name="text">
        <string>History</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/history.png</normaloff>../images/history.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Contacts_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background-color: rgba(255, 255, 255, 255);
font: 12pt &quot;Ubuntu&quot;;
border: 0px;}
QPushButton:focuspressed {background-color: rgba(255, 255, 255, 255);
gridline-color: rgb(198, 70, 0);
font: 12pt &quot;Ubuntu&quot;;
color:black;
border: 0px; }
 QPushButton:pressed {background-color: rgba(150, 150, 150, 255);
gridline-color: rgb(198, 70, 0);
font: 12pt &quot;Ubuntu&quot;;
color:black;
border: 0px; }</string>
       </property>
       <property name="text">
        <string>Contacts</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/contacts.png</normaloff>../images/contacts.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
